Kernel/DoubleBuffer.cpp

```diff
@@ -77,4 +77,24 @@ ssize_t DoubleBuffer::read(UserOrKernelBuffer& data, size_t size)
     return (ssize_t)nread;
 }
 
+ssize_t DoubleBuffer::peek(UserOrKernelBuffer& data, size_t size)
+{
+    if (!size || m_storage.is_null())
+        return 0;
+    VERIFY(size > 0);
```

Comment:
`VERIFY` doesn't do anything here, and `size > 0` wouldn't make sense when `m_storage.is_null()`.
```suggestion
    if (!size || m_storage.is_null())
        return 0;
```